cluster-up/cluster/kind-1.23-sriov/certcreator/certlib/selfsign.go

```diff
@@ -0,0 +1,115 @@
+package certlib
```

Comment:
Please remove the whole folder of cert creator, we don't use it for long IIRC

Reply:
I see multiple references to this directory:
https://github.com/kubevirt/kubevirtci/pull/897/files#diff-ba18cea87edeee1e4c652aaa05ccd4ac663d4b4545d892c9a1db306c6b5b9bb0R6

https://github.com/kubevirt/kubevirtci/pull/897/files#diff-ba18cea87edeee1e4c652aaa05ccd4ac663d4b4545d892c9a1db306c6b5b9bb0R182-R195

Reply:
but we dont deploy sriov_operator.sh anymore, not maintain it
so it use very old version
we can remove that as well please imo

Reply:
I suggest we remove it in a follow-up PR.
I don't think it helps the reviewer to see that removal in this PR.

Reply:
Imo an additional commit that removes it can be clean and easy enough to review,
I don't see why to take the files with us again and again.
but whatever you prefer

Reply:
I prefer to handle this change in a follow up PR.
I think there are more leftovers, for example:
https://github.com/kubevirt/kubevirtci/pull/897/files#diff-c5f5c4a7c70b52eb6c6561e5629101c50defa39f231960e5a5d4751f4152c82dR1

Reply:
I honestly don't see a reason to post phone it, it will make this PR not clean enough
It should be pretty trivial to clean all needed, as it is not used
maybe worth to at least try first ?

we can also wait to see more opinions

Reply:
I think there are more leftovers that we are not aware of.
Starting a cleanup operation could be an adventure that I don't want to get into at the moment.
Additionally I think that there could be multiple small improvements that could be made in order to improve the code.
But, they all could be done in follow-up PRs.

Reply:
The sriov operator scope should be few files that you just delete and forgot
for example the script / yamls and the cert lib
imo it is right to at least try it, (and if you see it is adventure drop it) but again this is just my 2 cents
rest of the cleaning can be done of course on follow PR

Reply:
I would remove what I can, but I won't insist that it will be part of this PR.